OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs

```diff
@@ -256,6 +262,26 @@ public void Activate(Order order)
 			if (Info.OneShot)
 				PrerequisitesAvailable(false);
 		}
+
+		bool HasSufficientFunds(SupportPower power, bool activate = false)
+		{
+			if (power.Info.Cost != 0)
```

Comment:
TakeCash will behave badly if a negative cost is passed in, so this should be > 0.